web/components/new-contract-panel.tsx

```diff
@@ -307,6 +340,7 @@ export function NewContractPanel(props: {
                 'Multi choice': 'MULTIPLE_CHOICE',
                 'Free response': 'FREE_RESPONSE',
                 Numeric: 'PSEUDO_NUMERIC',
+                Stonk: 'STONK',
```

Comment:
I think calling it a Stonk instead of a stock is actually a mistake. It would be one thing if our users actually called them that themselves, but they don't. 95% of people are just going to assume it's a weird typo.